helm/idol-library-example-test/values.schema.json

```diff
@@ -211,8 +211,12 @@
                 "ingress": {
                     "type":"object",
                     "properties":{
+                        "path": {
+                            "$ref": "#/$defs/required_string"
+                        }
                     },
                     "required":[
+                        "path"
                     ],
```

Comment:
`path` should go in `#/$defs/ingress/Common`, as should `servicePath`. Could put `indexPath` here as it's not valid for all aci servers

Reply:
And what gets done here will need reflecting in the other charts